playground/components/framework-switcher.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+
+import { Frameworks } from '@/types';
+import { useSandpack } from '@codesandbox/sandpack-react';
+import { Dispatch, SetStateAction } from 'react';
+
+export function FrameworkSwitcher({
+  setFramework,
```

Comment:
I would not drill this down as props. I would have it come from a context.

Reply:
Good job folks!!